### PR TITLE
[Stack 773] - Refactor of common module

### PR DESCRIPTION
**Pull request by omkartelee01**
**closes**
[STACK-773](https://a10networks.atlassian.net/browse/STACK-773)

### CODE REVIEW

File: a10_octavia/common/a10constants.py [hunk 1]
```diff
@@ -19,23 +19,50 @@
...
 
+BACKUP_AMPHORA_PLUG = 'backup-amphora-plug'
+BACKUP_CONNECTIVITY_WAIT = 'backup-connectivity-wait'
+BACKUP_ENABLE_INTERFACE = 'backup-enable-interface'
 MARK_VTHUNDER_MASTER_ACTIVE_IN_DB = 'mark-vthunder-master-active-in-db'
 MARK_VTHUNDER_BACKUP_ACTIVE_IN_DB = 'mark-vthunder-backup-active-in-db'
 GET_BACKUP_VTHUNDER_BY_LB = 'get-backup-vthunder-by-lb'
 CREATE_HEALTH_MONITOR_ON_VTHUNDER_MASTER = 'create-health-monitor-on-vthunder-master'
 HANDLE_SESS_PERS = 'handle-session-persistence-delta-subflow'
+GET_LOADBALANCER_FROM_DB = 'Get-Loadbalancer-from_db'
```
Comment (hthompson6) [inline a10_octavia/common/a10constants.py]: This should be: ```get-loadbalancer-from-db````

File: a10_octavia/common/a10constants.py [hunk 2]
```diff
@@ -19,23 +19,50 @@
...
+GET_BACKUP_LOADBALANCER_FROM_DB = 'get-backup-loadbalancer-from-db'
+CONFIGURE_VRRP_FOR_MASTER_VTHUNDER = 'configure-vrrp-for-master-vthunder'
+CONFIGURE_VRRP_FOR_BACKUP_VTHUNDER = 'configure-vrrp-for-backup-vthunder'
+CONFIGURE_VRID_FOR_MASTER_VTHUNDER = 'configure-vrid-for-master-vthunder'
+CONFIGURE_VRID_FOR_BACKUP_VTHUNDER = 'configure-vrid-for-backup-vthunder'
+CONFIGURE_VRRP_SYNC = 'configure-vrrp-sync'
+WAIT_FOR_MASTER_SYNC = 'wait-for-master_sync'
```
Comment (hthompson6) [inline a10_octavia/common/a10constants.py]: Both of these should be `wait-for-master-sync` and `wait-for-backup-sync`

Comment (omkartelee01) [reply a10_octavia/common/a10constants.py]: Correction completed

File: a10_octavia/common/a10constants.py (hunk 1 above)
Comment (omkartelee01) [reply a10_octavia/common/a10constants.py]: Correction completed

File: requirements.txt [hunk 3]
```diff
@@ -1,4 +1,5 @@
 acos-client>=1.4.6		#
+httplib2>=0.12.1                # Python HTTP client
```
[the remark this answers is not in the document]
Comment (omkartelee01) [reply requirements.txt]: Correction completed

File: .travis.yml [hunk 4]
```diff
@@ -2,7 +2,7 @@ language: python
 jobs:
   include:
   - name: "2.7 Unit Tests"
-    python: "2.7"
+    python: "2.7.17"
```
[the remark this answers is not in the document]
Comment (omkartelee01) [reply .travis.yml]: Correction completed

**Pull request merged**